ext/readme.md

```diff
@@ -1,7 +1,7 @@
 ## ext
  Status  |  Package  |  Version / Commit  | Usage | Notes
 :------: | :-------: | :----------------: | :---- | :----
-:: | `adba` | pymedusa/[119b9d3](https://github.com/pymedusa/adba/tree/119b9d30a30feb97bcea5e1dc742c82340300091) | **`medusa`** | -
+:: | `adba` | pymedusa/[119b9d3](https://github.com/pymedusa/adba/tree/70bc381a75e20e1813c848c1edb7c6f16987397e) | **`medusa`** | -
```

Comment:
```diff
- pymedusa/[119b9d3](https://github.com/pymedusa/adba/tree/70bc381a75e20e1813c848c1edb7c6f16987397e)
+ pymedusa/[70bc381](https://github.com/pymedusa/adba/tree/70bc381a75e20e1813c848c1edb7c6f16987397e) 
```

Maybe I should just change the link to say `pymedusa/[commit]` instead, it would be easier to update...